src/backend/web/static/swagger/api_v3.json

```diff
@@ -5846,6 +5846,12 @@
       "Match_Score_Breakdown_2015": {
         "type": "object",
         "properties": {
+          "required": [
```

Comment:
I think this is tabbed wrong. It should be outside of "properties" right?

Reply:
whoops